be/test/CMakeLists.txt

```diff
@@ -117,8 +116,6 @@ set(EXEC_FILES
         io/array_input_stream_test.cpp
         ./io/fd_output_stream_test.cpp
         io/fd_input_stream_test.cpp
-        ./io/s3_output_stream_test.cpp
-        io/s3_input_stream_test.cpp
```

Comment:
it seems this should not be removed, you can ask zhuming.

Reply:
It's preserved in following section.

```
if (WITH_AWS)
    set(EXEC_FILES ${EXEC_FILES} ./io/s3_output_stream_test.cpp ./io/s3_input_stream_test.cpp
        ./env/env_s3_test.cpp)
endif()
```